azurerm/resource_arm_front_door.go

```diff
@@ -198,7 +199,7 @@ func resourceArmFrontDoor() *schema.Resource {
 											string(frontdoor.HTTPSOnly),
 											string(frontdoor.MatchRequest),
 										}, false),
-										Default: string(frontdoor.MatchRequest),
+										Default: string(frontdoor.HTTPSOnly),
```

Comment:
This is a breaking change? we can't do this until 2.0

Reply:
Reverted default value change to avoide the breaking change, will open issue to update this in the 2.0 timeframe.